test/test_UPF/main.cpp

```diff
@@ -101,16 +102,20 @@ class UPFSimulation : public SIS
 };
 
 
-int main()
+int main(int argc, char* argv[])
 {
     std::cout << "Running an unscented particle filter on a simulated target." << std::endl;
-    std::cout << "Data is logged in the test folder with prefix testUPF." << std::endl;
+
+    const bool write_to_file = (argc > 1 ? std::string(argv[1]) == "ON" : false);
+    if (write_to_file)
+        std::cout << "Data is logged in the test folder with prefix testSIS." << std::endl;
```

Comment:
I think it should be `... with prefix testUPF`.

Reply:
Done!